tools/generator.sh

```diff
@@ -315,11 +323,26 @@ main() {
     inputFile=$(mktemp -t projects.json.XXXXXXXXXX)
     finish() {
         rm -f "$inputFile"
+        rm -f "$cookieJar"
     }
     trap finish EXIT
 
+    if [ "$includePrivate" = 1 ]; then
+      echo -n "Username: "
+      read -r username
+      echo -n "Password: "
+      read -rs pw
+      curl -X POST -c "$cookieJar" \
+        "$serverRoot"/login \
+        --fail --silent --show-error -o /dev/null \
+        -d '{"username":"'"$username"'","password":"'"$pw"'"}' \
```

Comment:
How about using `jq` to construct this JSON document, in case there are "s in the password?